options/locale/locale_en-US.ini

```
@@ -681,7 +681,7 @@ choose_new_avatar = Choose new avatar
 update_avatar = Update Avatar
 delete_current_avatar = Delete Current Avatar
 uploaded_avatar_not_a_image = The uploaded file is not an image.
-uploaded_avatar_is_too_big = The uploaded file has exceeded the maximum size.
+uploaded_avatar_is_too_big = The uploaded file size (%d KB) exceeds the maximum size (%d KB).
```

```suggestion
uploaded_avatar_is_too_big = The uploaded file size (%d KiB) exceeds the maximum size (%d KiB).
```

If you divide by 1024, KiB is the correct unit.